mooncake-transfer-engine/src/transfer_engine.cpp

```diff
@@ -57,6 +57,31 @@ int TransferEngine::freeEngine() {
     return 0;
 }
 
+int TransferEngine::submitTransfer(BatchID batch_id,
+                                   const std::vector<TransferRequest> &entries) {
+    // TODO: return Status in public function.
```

Comment:
could you explain the plan to reach it? seems we just use Status in internal APIs?

Reply:
The plan is to replace the return value in public function, so as the return value will not conflict with each other within cross components. The reason to set TODO here is because the convert job is non-trivial, and may even impact the vLLM code in Mooncake upstream. So I send out this PR for the review comments.

Reply:
okay, that make sense, the exposed C APIs need to take care too.

Reply:
okay, I'll proceed with this PR.